core/SConscript.kernel

```diff
@@ -3,6 +3,7 @@
 
 import os
 import tools, models
+import random
```

Comment:
not needed in kernel anymore

Reply:
https://github.com/trezor/trezor-firmware/commit/883f54f367646bb095164447746da9e1a3c9e039